### PR TITLE
Added multipart upload with bucket policy enabled TC

### PR DESCRIPTION
**Pull request by udaysk23**
Added multipart upload with bucket policy enabled TC

Following is the link of successful run
http://magna002.ceph.redhat.com/cephci-jenkins/cephci-run-QCFLKD
# Description

Please include Automation development guidelines. Source of Test case - New Feature/Regression Test/Close loop of customer BZs

# Checklist:

- [x] Create a test case in Polarion reviewed and approved.
- [ ] Create a design/automation approach doc. Optional for tests with similar tests already automated.
- [ ] Review the automation design
- [x] Implement the test script and perform test runs
- [x] Submit PR for code review and approve
- [x] Update Polarin Test with Automation script details and update automation fields
- [ ] If automation is part of Close loop, update BZ flag qe-test_coverage “+” and link Polarion test

### CODE REVIEW

File: suites/pacific/rgw/tier_1_extn_object.yaml [hunk 1]
```diff
@@ -116,4 +116,14 @@ tests:
...
+      name: Multipart upload with Bucket policy enabled
+      desc: Perform multipart upload with Bucket policy enabled
+      polarion-id: CEPH-83574538
+      module: sanity_rgw.py
+      config:
+        script-name: test_bucket_policy_ops.py
+        config-file-name: test_bucket_policy_with_multipart_upload.yaml
+        timeout: 500
+        
+
```
Comment (psathyan) [inline suites/pacific/rgw/tier_1_extn_object.yaml:129]: Please remove empty lines

**Comment by rakeshgm** [comment]
@udaysk23 couple of things, 
1. Please add nautilus logs and add the test case in the nautilus suite 
2. Please update the checklist activities in this PR

**Review by viduship — DISMISSED** [review]
Looks good to me.

**Comment by rakeshgm** [comment]
@udaysk23 rebase or fix conflicts. and please makr the checklist.

**Comment by udaysk23** [comment]
Following is the successful run on nautilus branch
http://magna002.ceph.redhat.com/cephci-jenkins/cephci-run-HLCILR

**Review by rakeshgm — APPROVED** [review]

**Pull request merged**